chromadb/config.py

```diff
@@ -91,6 +92,11 @@
 DEFAULT_DATABASE = "default_database"
 
 
+class APIVersion(str, Enum):
```

Comment:
Should this be in server/ ?

Reply:
I think this will cause a circular import

Reply:
Checked and it does

Reply:
Ah my b